components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityTenantUtil.java

```diff
@@ -164,13 +167,16 @@ public static RegistryService getRegistryService() {
         return registryService;
     }
 
+    public static void setRegistryService(RegistryService registryService) {
```

Comment:
Add a new line after public methods. Check all the places. Let's add method comments for public methods.

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/3767